examples/platform-specific/cc26xx/cc26xx-ble-client-demo/project-conf.h

```diff
@@ -0,0 +1,79 @@
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE
+ * COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * \author
+ *    Michael Spoerk <michael.spoerk@tugraz.at>
+ */
+/*---------------------------------------------------------------------------*/
+#ifndef PROJECT_CONF_H_
+#define PROJECT_CONF_H_
+
+/*---------------------------------------------------------------------------*/
+/* Disable button shutdown functionality */
+#define BUTTON_SENSOR_CONF_ENABLE_SHUTDOWN      0
+/*---------------------------------------------------------------------------*/
+/* Change to match your configuration */
+#define BOARD_CONF_DEBUGGER_DEVPACK             1
+/*---------------------------------------------------------------------------*/
+#define PACKETBUF_CONF_SIZE                   1280
+#define QUEUEBUF_CONF_NUM                       1
+#define UIP_CONF_BUFFER_SIZE                  1280
+
+#define CC26XX_CONF_RADIO_MODE            RADIO_MODE_BLE
+#define NETSTACK_CONF_RADIO                 ble_cc2650_driver
+#define MAKE_MAC              MAKE_MAC_OTHER
+#define NETSTACK_CONF_MAC                   ble_l2cap_driver
+
+#define LOG_CONF_LEVEL_MAC        LOG_LEVEL_INFO
+#define LOG_CONF_LEVEL_6LOWPAN      LOG_LEVEL_WARN
+#define LOG_CONF_LEVEL_IPV6       LOG_LEVEL_WARN
+#define LOG_CONF_LEVEL_TCPIP      LOG_LEVEL_WARN
+
+/* BLE L2CAP settings */
+#define BLE_CONF_DEVICE_NAME          "TI CC26xx device"
+#define BLE_CONF_ADV_INTERVAL         25
+
+#define MAKE_NET              MAKE_NET_IPV6
+#define MAKE_ROUTING            MAKE_ROUTING_NONE
```

Comment:
These flags should be in the makefile: https://github.com/contiki-ng/contiki-ng/wiki/The-Contiki%E2%80%90NG-configuration-system